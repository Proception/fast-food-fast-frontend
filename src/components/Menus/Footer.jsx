import React, { Component } from 'react';
import { NavLink } from 'react-router-dom';

export default class Footer extends Component {
  render() {
    return (
    <footer>
        <p>Fast Food Fast Copyright &copy; 2018</p>
    </footer>
    )
  }
}
