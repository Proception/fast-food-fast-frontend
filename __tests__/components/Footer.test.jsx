import React from 'react';
import { shallow } from 'enzyme';
import Footer from '../../src/components/Menus/Footer.jsx';

let wrapper;
beforeEach(() => {
  wrapper = shallow(<Footer/>);
});
describe('Footer Component', () => {
  it('should match snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
