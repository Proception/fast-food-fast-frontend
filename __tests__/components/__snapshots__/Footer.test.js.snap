// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Component should match snapshot 1`] = `
<footer>
  <p>
    Fast Food Fast Copyright Â© 2018
  </p>
</footer>
`;
